src/User.php

```diff
@@ -5120,7 +5120,10 @@ public function showItems($tech)
                     $itemtable = getTableForItemType($itemtype);
                     $iterator_params = [
                         'FROM'   => $itemtable,
-                        'WHERE'  => ['OR' => $group_where]
+                        'WHERE'  => [
+                            'entities_id' => $this->getEntities(),
+                            'OR'          => $group_where
+                        ]
```

Comment:
You can see a user as long as you have access to, at least, one of the same entities as the user itself. It does not mean that you have access to all the entities that the user can access. Therefore, filtering on `$this->getEntities()` could still shows items you are not supposed to see.

Restricting to session active entities is probably a safer solution. I did not check if it does the job, but it would correspond to something like that:
```suggestion
                        'WHERE'  => [
                            'OR'          => $group_where
                        ] + getEntitiesRestrictCriteria($itemtable, '', '', true)
```

Reply:
Currently, if the listed item is not in an available entity (corresponding to the session's active entities), it is not clickable. _The item name is a link only if it is in an accessible entity_.

If we filter on the session's active entities, in some cases we may list items to which the user does not have access. I think this is what the tab is for: to list the items assigned to the user (via his groups) and to which he has access.

I can use both, but I don't think `getEntitiesRestrictCriteria` should replace `getEntities`.